plugins/insomnia-plugin-default-headers/src/default-headers.js

```diff
@@ -12,7 +12,12 @@ module.exports = function(context) {
       continue;
     }
 
-    context.request.setHeader(name, value);
-    console.log(`[header] Set default header ${name}: ${value}`);
+    if (value==="null") {
+      context.request.setHeader(name, null);
```

Comment:
Might be better to use [`removeHeader`](https://github.com/Kong/insomnia/blob/0178533e5284cbcaff2c39a4ba2d560b16422e4c/packages/insomnia-app/app/plugins/context/request.ts) here

```suggestion
      context.request.removeHeader(name);
```